nerdlets/util/graphqlbuilders.js

```diff
@@ -127,7 +127,7 @@ export function buildToggleRuleQuery(accountId, ruleId, enabled) {
   const mutation = `mutation {
   eventsToMetricsUpdateRule(updates: {accountId: ${accountId}, enabled: ${enabled}, ruleId: ${ruleId}}){
     failures {
-      errors {
+      error {
```

Comment:
There weren't any graphql deprecations, this should still be `errors`
```suggestion
      errors {
```